flax/__init__.py

```diff
@@ -15,5 +15,5 @@
 """Flax API."""
 
 # Allow `import flax`; `flax.nn.[...]`, and the same for `flax.optim.[...]`
-from . import nn
+from flax import nn
```

Comment:
Let's not make aesthetic changes in this CL. (If we think avoiding `from .` is desirable, we can discuss in a separate CL that only changes all of those occurrences.)